### PR TITLE
VEP properties json

### PR DESCRIPTION
**Pull request by cseed**
- convert the VEP configuration file to JSON
 - let the conf file fully specify the command line, environment and output schema

This is breaking change.  I updated the VEP config files on the cloud:

```
$ gsutil ls gs://hail-common/vep/vep/*.json
gs://hail-common/vep/vep/vep81-gcloud.json
gs://hail-common/vep/vep/vep85-gcloud.json
gs://hail-common/vep/vep/vep92-GRCm38-gcloud.json
```

But pipelines will have to change to use them.  I tested GRCm38, but not the other ones.  @konradjk, would you be up for testing the standard usage?  (We really need to add automated VEP tests.)

### CODE REVIEW

**Comment by cseed** [comment]
Oops, I still have to update the `vep` docs.

**Comment by cseed** [comment]
Updated docs.  Should be good to go.

**Comment by catoverdrive** [comment]
looks fine to me. @konradjk can you approve once tested? thanks!

**Comment by tpoterba** [comment]
what's my review responsibility here? Standard review process? How does this interact with Jon setting up tests?

**Comment by cseed** [comment]
@tpoterba @jbloom22 I think you're off the hook for the moment, we're waiting on Jon for tests.  Once this goes in, I'll rebase/revise mine.

**Comment by cseed** [comment]
@tpoterba OK, this is ready for review.  It passes with Jon's VEP test setup:

```
Checking 'hl.vep' replicates on 'gs://hail-common/vep/vep/vep_examplars/vep_35d9e30.mt'
2018-08-10 22:02:28 Hail: INFO: vep: annotated 1196 variants
TEST PASSED
```

invoked as:

```
actual = hl.vep(expected.select(), 'gs://hail-common/vep/vep/vep85-gcloud.json')
```

I will prepare a discuss post for when it is ready to merge.

File: src/main/scala/is/hail/methods/VEP.scala [hunk 1]
```diff
@@ -302,7 +124,14 @@ object VEP {
                       fatal(s"VEP output variant ${ VariantMethods.locusAllelesToString(vepLocus, vepAlleles) } not found in original variants.\nVEP output: $s")
                   }
                 } else {
-                  val a = JSONAnnotationImpex.importAnnotation(JsonMethods.parse(s), vepSignature)
+                  val jv = try {
+                    JsonMethods.parse(s)
+                  } catch {
+                    case _: JsonParseException =>
+                      log.warn(s"vep: failed to parse json: $s")
```
Comment (cseed) [inline src/main/scala/is/hail/methods/VEP.scala]: FYI I added this to handle some malformed JSON generated by VEP when annotation mouse genomes.

**Pull request merged**